screens/LoginSignUp.js

```diff
@@ -0,0 +1,188 @@
+import React, {useState} from 'react';
+import {KeyboardAwareScrollView} from 'react-native-keyboard-aware-scroll-view';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  TouchableOpacity,
+  TouchableHighlight,
+  TextInput,
+  Linking,
+  KeyboardAvoidingView,
+} from 'react-native';
+
+const LoginSignUp = props => {
+  const [isLogin, setVisible] = useState(true);
+  return (
+    <KeyboardAwareScrollView
+      contentContainerStyle={{flexGrow: 1}}
+      style={styles.removePadding}>
+      <View style={styles.appContainer}>
+        <View style={[styles.alignImage, styles.whiteBackground]}>
+          <View style={[styles.whiteBackground, styles.selfCenter]}>
+            <Image source={require('../assets/images/loginImage.png')} />
+          </View>
+        </View>
+
+        <View style={[styles.tabContainer, styles.whiteBackground]}>
+          <TouchableHighlight
+            onPress={() => {
```

Comment:
It's a good practice to remove business logic outside of view so keep these onpress handlers outside of JSX and pass it like `handleLogin or handleSignup`